### PR TITLE
Fixed launcher threading - Fuse.iOS - OpenURL

### PR DESCRIPTION
**Pull request by AndrewEQ**
Fixed threading issue for the iOS url launcher

### CODE REVIEW

File: Source/Fuse.iOS/Fuse.iOS.unoproj [hunk 1]
```diff
@@ -2,9 +2,6 @@
   "Copyright": "Copyright (c) Fuse Open 2018",
   "Description": "Native UI and bootstrapping for iOS",
   "Publisher": "Fuse Open",
-  "Package": {
-    "ProjectUrl": "https://fuseopen.com"
-  },
```
Comment (kusma) [inline Source/Fuse.iOS/Fuse.iOS.unoproj]: This is already like this on master, perhaps you could rebase on master to make this go away? :)

**Comment by AndrewEQ** [comment]
oops, my bad

File: Source/Fuse.iOS/Fuse.iOS.unoproj (hunk 1 above)
Comment (AndrewEQ) [reply Source/Fuse.iOS/Fuse.iOS.unoproj]: strange where that came from cos I originally rebased, anyhoo, sorted it out

**Comment by AndrewEQ** [comment]
Cool, how can we get travis to run again?

**Comment by kusma** [comment]
I believe it's already re-running, something seems to be wrong with the reporting:
https://travis-ci.org/fuse-open/fuselibs/builds/586046406

**Comment by kusma** [comment]
@AndrewEQ: I don't know why it's stuck. The new build passed, but it's not tagged correctly. Probably a glitch due to the closing of the PR and poor timing.

Perhaps you could do a `git commit --amend --reset-author` to give the tip of the branch a new timestamp, and then re-push?

**Comment by AndrewEQ** [comment]
ok, lemme try get out and give it a push

**Pull request merged**